src/engine/uct/node/mod.rs

```diff
@@ -146,21 +148,40 @@ impl Node {
                 .iter()
                 .map(|&m| Node::new(m, self.config))
                 .collect();
+            if self.children.len() <= (game.size() * game.size() / 10) as usize {
+                let player = game.last_move().color().opposite();
```

Comment:
How about adding `next_player()` to `Game`? It would just return the `board.next_player()`. That looks cleaner.